install/installer/pkg/components/server/deployment.go

```diff
@@ -173,8 +173,7 @@ func deployment(ctx *common.RenderContext) ([]runtime.Object, error) {
 			},
 			Spec: appsv1.DeploymentSpec{
 				Selector: &metav1.LabelSelector{MatchLabels: labels},
-				// todo(sje): receive config value
-				Replicas: pointer.Int32(1),
+				Replicas: common.Replicas(ctx, Component),
```

Comment:
Once we agreed that this makes sense we should replicate it to `StatefulSet`s as well.

Reply:
Every deployment/statefulset that needs to be able to specify its replica count via `experimental` config  will need this change - `Replicas` is hard-coded to `1` everywhere. This change to `server` is just to illustrate the pattern.